e2e/tenant_control_plane_ready_test.go

```diff
@@ -0,0 +1,88 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+
+	kamajiv1alpha1 "github.com/clastix/kamaji/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
`k8s.io/apimachinery/pkg/apis/meta/v1` must be in the second block since it's a dependency, the third one is reserved for internal packages (`github.com/clastix/kamaji`)